types/bech32/bech32.go

```diff
@@ -18,7 +18,7 @@ func ConvertAndEncode(hrp string, data []byte) (string, error) {
 
 // DecodeAndConvert decodes a bech32 encoded string and converts to base64 encoded bytes.
 func DecodeAndConvert(bech string) (string, []byte, error) {
-	hrp, data, err := bech32.Decode(bech)
+	hrp, data, err := bech32.Decode(bech, 250)
```

Comment:
Pretty sure we have 90 for a reason. But it won't work with multisigs. I'm not familiar with any issues that could arise if we change it.

/cc @zmanian

Reply:
Yeah, 90 is from the bech32 specs and it was hard coded in the bech32 library.

In practice cosmos-sdk uses longer addresses, mainly for multisig public keys, and the 90 limit creates bugs (e.g. in Enigma people are using multisigs and thus we cannot export the state and validate it if we want to hard fork)